### PR TITLE
add the section for learning objectives and content 

### LINKED ISSUE #30: Describe learning objectives and content
Short (~200 word) description of learning objectives and content. Emphasize modularity of lectures, that while this is packaged as a course, each lecture is basically independent. Possibly outline which lectures go together? (Or maybe that should just be in the README in the course)

### PR DESCRIPTION
**Pull request by SaraMati**
closes #30

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -77,7 +77,40 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+The overarching objective of the course is to teach quantitative research
+methods that are reproducible and collaborative. Each lecture is independent and
+have specific learning objectives. First lecture starts with giving students an
+overview of the capabilities of R such as more efficient quantitative analysis
+of data compared to manual handling of spreadsheets, programming in R, and
+integrating analyses with generating reports using R Notebook (markdown syntax).
+Second lecture introduces fundamental elements in programming with R such as
+vectors, data frames, basic operations and use of functions. The third lecture
+intorduces the concept of exploratory analysis of datasets with a focus on using
```
Comment (aays) [inline paper.md]: s/intorduces/introduces/

File: paper.md [hunk 2]
```diff
@@ -77,7 +77,40 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+approach, its benefits over traditional inferential statistics, and model
+selection criteria such as Akaike’s Information Criterion. Lecture twelve is to
+learn population models and using of differential equations such to analyze
+system dynamics. Lecture thirteen focuses on time series data and fitting
+numerical models to such datasets. Lecture fourteen introduces the necessity to
+acquire a scientific approach, the steps involved and integrating such approach
+with group dynamics and in assigning roles and tasks. In this lecture students
+form groups for the final course project. Lecture fifteen focuses on
+reproducible and collaborative research tools such as Git version control and
+Github platform. Each project group make their Github repository and the team
```
Comment (aays) [inline paper.md]: Maybe 'Each group works within a GitHub repository'?

File: paper.md [hunk 3]
```diff
@@ -77,7 +77,40 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+members learn the basics of documenting the progress of the project and
+collaborate using these tools. Lecture sixteen, finishes the course with the
```
Comment (aays) [inline paper.md]: s/Lecture sixteen, finishes/Lecture sixteen finishes/

File: paper.md [hunk 4]
```diff
@@ -77,7 +77,40 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+The overarching objective of the course is to teach quantitative research
+methods that are reproducible and collaborative. Each lecture is independent and
+have specific learning objectives. First lecture starts with giving students an
```
Comment (aays) [inline paper.md]: s/First lecture/The first lecture/

File: paper.md [hunk 5]
```diff
@@ -77,7 +77,40 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+overview of the capabilities of R such as more efficient quantitative analysis
+of data compared to manual handling of spreadsheets, programming in R, and
+integrating analyses with generating reports using R Notebook (markdown syntax).
+Second lecture introduces fundamental elements in programming with R such as
```
Comment (aays) [inline paper.md]: /s/Second lecture/The second lecture/

File: paper.md [hunk 6]
```diff
@@ -77,7 +77,40 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+acquire a scientific approach, the steps involved and integrating such approach
+with group dynamics and in assigning roles and tasks. In this lecture students
+form groups for the final course project. Lecture fifteen focuses on
+reproducible and collaborative research tools such as Git version control and
+Github platform. Each project group make their Github repository and the team
+members learn the basics of documenting the progress of the project and
+collaborate using these tools. Lecture sixteen, finishes the course with the
+topic of reproducibility. We learn the barriers, and the beneficial approaches
+such as the use of metadata and generating comprehensive manuscript in
+R-markdown.
```
Comment (aays) [inline paper.md]: s/R-markdown/R Markdown/

File: paper.md [hunk 7]
```diff
@@ -77,7 +77,40 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+dplyr package. The objective for lecture four is to learn the advantage of using
+summary statistics and data visualization using tidyverse packages. Lecture five
+focuses on approaches for cleaning up and preprocessing of raw data. Lecture
+seven introduces basic descriptive and inferential statistical techniques such
+as linear models. Lecture eight focuses on linear mixed effects models to
+analyze the fixed and random effects in nested exploratory data. Lecture nine
+introduces randomization tests and teaches simulating datasets and applying such
+tests on simulated and read data. Lecture ten's objective is to learn
+multivariate statistics with a focus on principle component analysis and
+multivariate analysis of variance. Lecture eleven introduces model selection
```
Comment (aays) [inline paper.md]: s/introduces model selection approach/introduces the model selection approach/

File: paper.md [hunk 8]
```diff
@@ -76,4 +76,41 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
 
 # Main Body
 
+## learning objectives and content
+The overarching objective of the course is to teach quantitative research
+methods that are reproducible and collaborative. Each lecture is independent and
+have specific learning objectives. First lecture starts with giving students an
```
Comment (mbonsma) [inline paper.md]: `has specific learning objectives`

**Comment by mbonsma** [comment]
If we're going to describe each lecture individually, I would make this a bullet-point list instead, grouped into sets of lectures that go together. (@aays probably has opinions on this since he's doing the same in the README.)

**Comment by mbonsma** [comment]
Cite packages that are mentioned and make them `code font`.

File: paper.md [hunk 9]
```diff
@@ -76,4 +76,41 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+seven introduces basic descriptive and inferential statistical techniques such
+as linear models. Lecture eight focuses on linear mixed effects models to
+analyze the fixed and random effects in nested exploratory data. Lecture nine
+introduces randomization tests and teaches simulating datasets and applying such
+tests on simulated and read data. Lecture ten's objective is to learn
+multivariate statistics with a focus on principle component analysis and
+multivariate analysis of variance. Lecture eleven introduces model selection
+approach, its benefits over traditional inferential statistics, and model
+selection criteria such as Akaike’s Information Criterion. Lecture twelve is to
+learn population models and using of differential equations such to analyze
```
Comment (mbonsma) [inline paper.md]: `using differential equations to analyze`

File: paper.md [hunk 10]
```diff
@@ -76,4 +76,41 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+system dynamics. Lecture thirteen focuses on time series data and fitting
+numerical models to such datasets. Lecture fourteen introduces the necessity to
+acquire a scientific approach, the steps involved and integrating such approach
```
Comment (mbonsma) [inline paper.md]: `Lecture fourteen introduces the scientific method and integrating scientific approaches with group dynamics`

File: paper.md [hunk 11]
```diff
@@ -76,4 +76,41 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+with group dynamics and in assigning roles and tasks. In this lecture students
+form groups for the final course project. Lecture fifteen focuses on
+reproducible and collaborative research tools such as Git version control and
+Github platform. Each project group make their Github repository and the team
```
Comment (mbonsma) [inline paper.md]: and the GitHub platform

**Comment by aays** [comment]
I've grouped the lectures into:

1. Introductory R (Lectures 1-5)
2. Statistical analysis (Lectures 6-12)
3. Reproducible science (Lectures 13-15)

Does this categorization look good? If so, we could add the lecture descriptions under these with bullet points.

**Review by mbonsma — CHANGES_REQUESTED** [review]

**Comment by mbonsma** [comment]
> I've grouped the lectures into:
> 
>     1. Introductory R (Lectures 1-5)
> 
>     2. Statistical analysis (Lectures 6-12)
> 
>     3. Reproducible science (Lectures 13-15)
> 
> 
> Does this categorization look good? If so, we could add the lecture descriptions under these with bullet points.

Yeah, that looks good, although what do you think about separating out my lectures into their own section since they're pretty much completely independent from the others?

**Comment by aays** [comment]
@mbonsma Revised as follows:

1. Introductory R (Lectures 1-5)
    * Introduces students to the R programming language, with a focus on 
      data wrangling and visualization.
2. Statistical analysis (Lectures 6-10)
    * Introduces concepts such as regression, principal component analysis, and statistical models
3. Numerical modelling (Lectures 11-12)
    * Introduces students to numerically modelling populations in R
3. Reproducible science (Lectures 13-15)
    * Prepares students for project work period and introduces methods for reproducible science (GitHub, R Markdown)

**Comment by SaraMati** [comment]
I wrote "three main groups" in describing them, but then divided to the "four independent modules"  : 
The lectures are designed in three main groups: Programming in R and basic data wrangling and visualisations (lectures 1-5), exploratory data analysis, statistics and modelling (Lectures
6-12),  and reproducible science (Lectures 13-15).

**Review by mbonsma — APPROVED** [review]

**Pull request merged**